### DISCUSSION TITLE
Is there an option to set validation mode to 'onChange' to a particular validation rule.

### DISCUSSION THREAD

**Discussion by CodAffection** [react-hook-form/react-hook-form#10828] [Q&A] [answered]
I want to keep form validation `mode` as `onSubmit`, but want to validate an async validation rule like `onChange` mode? 
@bluebill1049

**Comment by Moshyfawn** [comment, marked as answer]
Use [the `trigger` method](https://react-hook-form.com/docs/useform/trigger) to trigger a specific field validation whenever you want.

_P.S. Please don't @ RHF maintainers directly; we see all issues and discussion and we'll get back to you when we have time 😉_
